### DISCUSSION TITLE
Welcome to pvfree Discussions!

### DISCUSSION THREAD

**Discussion by mikofski** [BreakingBytes/pvfree#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by markcampanelli** [comment]
Thanks @mikofski for this effort.

What do you think about putting IEC 61853-1 and -2 datasets on pvfree?

**Reply by mikofski** [reply]
For sure. Do you have a particular schema in mind? pvfree can store arrays, so could just store the matrix in one column rather than splitting them out.  Also do you have any idea where I can find trusted datasets to upload?
